frontend/talentsearch/src/js/components/pool/PoolAdvertisementPage.tsx

```diff
@@ -126,11 +135,36 @@ const PoolAdvertisement = ({ poolAdvertisement }: PoolAdvertisementProps) => {
     poolAdvertisement.nonessentialSkills,
   );
 
+  const [, executeMutation] = useCreateApplicationMutation();
+  const handleCreateApplication = () =>
+    executeMutation({ userId, poolId: poolAdvertisement.id })
+      .then((result) => {
+        if (result.data?.createApplication) {
+          navigate(paths.reviewApplication(result.data.createApplication.id));
+          toast(
+            intl.formatMessage({
+              defaultMessage: "Application created",
+              id: "U/ji+A",
+              description: "Application created successfully",
+            }),
+          );
```

Comment:
Update this to `toast.success` to match others.

```suggestion
          toast.success(
            intl.formatMessage({
              defaultMessage: "Application created",
              id: "U/ji+A",
              description: "Application created successfully",
            }),
          );
```